packages/db/src/services/DatabaseService.ts

```
@@ -39,217 +25,217 @@ export class DatabaseService {
-        return null;
-      }
+    if (typeof Rfid !== "string") {
+      throw new Error("Rfid must be a string");
+    }
 
-      const oldName = existingDriver.Name;
-      existingDriver.Name = name;
-      await this.driverRepo.save(existingDriver);
+    const existing = await prisma.driver.findUnique({
+      where: { rfid: Rfid },
+      select: { Name: true },
```

ESLint

[error] Expected object keys to be in ascending order. 'select' should be before 'where'.

(sort-keys/sort-keys-fix)